daml-assistant/integration-tests/src/Main.hs

```diff
@@ -389,6 +389,48 @@ packagingTests = testGroup "packaging"
             , "dependencies: [daml-prim, daml-stdlib]"
             ]
         withCurrentDirectory projDir $ callCommandQuiet "daml build"
+    , testCase "Dalf imports" $ withTempDir $ \projDir -> do
+        let genSimpleDalfExe
```

Comment:
It might be easier if you turn this into a `genrule` and then only reference the generated `dalf` here.

Reply:
moved it to a genrule